src/particles/particle_types.hpp

```diff
@@ -146,6 +154,20 @@ template <typename problem_t> using CICRadParticleIterator = amrex::ParIter<CICR
 // 1. For massive particles, velocity components start after mass
 // 2. Birth time, if existing, is always followed by death time
 
+// Function to parse particle parameters from input file
+// The 'inline' keyword allows this function to be defined in a header file without
+// causing multiple definition errors when the header is included in multiple source files.
+// It tells the linker that all instances of this function across different translation units
+// should be treated as the same function. This is a common pattern for small utility
+// functions defined in header files.
+inline void particleParmParse()
+{
+	// Parse particle parameters
+	const amrex::ParmParse pp("particles");
+	pp.query("param1", particle_param1);
+	pp.query("param2", particle_param2);
+}
+
```

Comment:
Is this a placeholder for future parameters that will replace these? Or can this code also go in `gravity_3d.cpp`?